packages/nextjs-template/components/DendronNoteGiscusWidget.tsx

```diff
@@ -32,8 +33,9 @@ export const DendronNoteGiscusWidget = ({
     const cleanGiscusConfig: GiscusProps = {
       ...giscusConfig,
       repo: repoString,
+      term: page,
```

Comment:
I thought having `mapping` as pathname would be sufficient. So is the pathname the same at the point of this widget being loaded?

Reply:
if that is the case we should remove the config `publishing.giscus.term`) as this does nothing at this point (or change this so that if `giscusConfig.term` exists, it uses that, and specify that if `term` is set in configs all giscus widgets will use the same term)

Reply:
i thought so too but doesn't look like it works. and yes - can you remove that from the config?